conan/tools/scm/git.py

```diff
@@ -4,6 +4,7 @@
 from conans.errors import ConanException
 from conans.util.files import mkdir
 from conans.util.runners import check_output_runner
+from conan.tools.scm import Version
```

Comment:
If the import is failing, could be some circular imports, try to put it local to the method.